helloworld/main.py

```diff
@@ -1,9 +1,20 @@
+#!/usr/bin/env python
```

Comment:
I'd prefer only having a shebang line and `if __name__ == '__main__'` stuff in one file used to run it, not adding it here and having multiple interchangeable options.